docs/additional-features/stress-testing.md

```diff
@@ -8,24 +8,46 @@ title: Network stress testing
 This guide assumes that:
 
 - You have a working Polygon-SDK network up and running
-- Both your JSON-RPC endpoints are reachable
-- For each of the sender accounts you use, its private key must be saved as an environment variable with the following format:
-  - PSDK_ADDRESS=PRIVATE_KEY
-- Each sender account must have the required funds on it to execute a single loadbot run
+- Both your JSON-RPC and GRPC endpoints are reachable
 
-e.g. to export the variable :
+## Overview
+
+The Polygon SDK loadbot is a helper that is meant to stress test the Polygon SDK cluster.
+
+Currently, it supports 2 modes:
+
+- `Transfer` - mode that does stress testing using fund-transfer transactions. **[Default]**.
+- `Deploy` - mode that deploys specified smart contracts with each transcation.
+
+### Transfer Mode
+
+The transfer mode assumes that there is a sender account that has initial funds to conduct the loadbot run.
+
+The sender's account address and private key need to be specified within the shell that's running the loadbot command.
```

Comment:
I think `setting in environment variables` is easy to understand.

Reply:
It does sound more clear 🙂 

Fixed 👍 

76c048d